Directory.Build.props

```diff
@@ -111,6 +111,7 @@
     <SystemFormatsAsn1Version>$(NetNineRuntimeVersion)</SystemFormatsAsn1Version>
     <SystemTextJsonVersion>$(NetNineRuntimeVersion)</SystemTextJsonVersion>
     <MicrosoftExtensionsDependencyInjectionVersion>$(NetNineRuntimeVersion)</MicrosoftExtensionsDependencyInjectionVersion>
+    <MicrosoftIdentityAbstractionsVersion>7.0.0</MicrosoftIdentityAbstractionsVersion>
```

Comment:
What happens if we reference IdentityAbstractions 7.0.0 in other frameworks? this should work too given the methods in Abstractions are defined only above 8.0.0?

I think we should systematically reference IdentityAbstractions 7.0.0

Reply:
Actually I checked. Just update line 91.
Let's just factorize in the common properties. No special case for net8.0 and net.9.0